package.json

```diff
@@ -35,12 +35,13 @@
   },
   "author": "Patrick Arlt <parlt@esri.com> (http://patrickarlt.com)",
   "contributors": [
-    "Patrick Arlt <parlt@esri.com> (http://patrickarlt.com)"
+    "Patrick Arlt <parlt@esri.com> (http://patrickarlt.com)",
+    "John Gravois <jgravois@esri.com> (http://johngravois.com)"
   ],
   "license": "Apache 2.0",
   "readmeFilename": "README.md",
   "dependencies": {
-    "leaflet": "^0.7.0",
-    "esri-leaflet": "^1.0.0-rc.5"
+    "leaflet": "0.7.3",
+    "esri-leaflet": "^1.0.0"
```

Comment:
@jgravois can you check if the tests are passing locally by pointing this at `git@github.com:Esri/esri-leaflet.git` and then running `npm cache clean && npm install && npm test`? If they passing locally I'll merge this and the tests should then pass once we release Esri Leaflet 1.0.0.